### PR TITLE
Cleanup standard skin

### PR DESCRIPTION
**Pull request by philipp-doblhofer**
Changes proposed in this pull request:
- Remove single-color images and gradients (replaced with CSS-code)
- Remove obviously unused image files

Reason for this pull request:
- Allow color customizations (corporate design) without changing image files

### CODE REVIEW

**Comment by kevinpapst** [comment]
@simonschaufi I'd like to have a look before merging!

**Comment by simonschaufi** [comment]
Wow, looks great!

One thing i noticed: In Firefox, I see a green line below the active tab:

grafik

Could you fix this?

The export module also contains a yellow bar at the top. You can change it the same way like the other boxes (like the one next to the start button)

The rest is fine! 🎉 great job!

File: skins/standard/menu.css [hunk 1]
```diff
@@ -2,70 +2,81 @@
...
-}
-
 .menu li.act span.bb,
-.menu li.ui-state-active span.bb{
-    background:url('grfx/g3_tab_black_middle.png');
+.menu li.ui-state-active span.bb {
+	border: 1px solid #404040;
+	border-top-left-radius: 5px;
+	border-top-right-radius: 5px;
+    height: 16px;
```
Comment (simonschaufi) [inline skins/standard/menu.css]: I think this should be 17px. 16px + 1px border = 17px but the others are 18px heigh.

**Comment by philipp-doblhofer** [comment]
No problem, I raised the height to 17px (but I couldn't reproduce it on my Firefox installation)
The box in the export extension is also replaced.

**Comment by kevinpapst** [comment]
@philipp-doblhofer  _this comment was hidden_

**Review by simonschaufi — APPROVED** [review]
looks great!

**Comment by kevinpapst** [comment]
How many browser and OS where tested? 
I don't mind merging, but to be honest: 66 changed files is a huge PR... less is more sometimes!
Nevertheless: thanks for your contribution @philipp-doblhofer

**Comment by simonschaufi** [comment]
> I don't mind merging, but to be honest: 66 changed files is a huge PR... less is more sometimes!

Well, its a big cleanup and for me that's totally fine.

**Comment by kevinpapst** [comment]
Well, different opinions are inspiring, aren't they ;-)

**Comment by simonschaufi** [comment]
Kevin, can I merge this one?

**Comment by kevinpapst** [comment]
If that was tested thoroughly, go ahead. 
@philipp-doblhofer are you available if someone finds some UI glitches?

**Comment by philipp-doblhofer** [comment]
@kevinpapst Sure, just mention me in the bug report and I should get an e-mail notification.

**Comment by kevinpapst** [comment]
@philipp-doblhofer 👍 thanks

**Pull request merged**